xtask/src/main.rs

```diff
@@ -429,36 +430,23 @@ fn run_shellcheck() -> Step {
 }
 
 fn run_clang_format(mode: FormatMode) -> Step {
-    match mode {
-        FormatMode::Check => Step::Single {
-            name: "clang format".to_string(),
-            command: Cmd::new(
-                "python3",
-                [
-                    "./third_party/run-clang-format/run-clang-format.py",
```

Comment:
Can we delete this file now then?

Reply:
Nothing to delete - it didn't exist. This was dead code.